CQUI.modinfo

```diff
@@ -54,6 +54,20 @@
     </Criteria>
   </ActionCriteria>
 
+    <!-- ============================== CQUI specific settings ============================== -->
+  <FrontEndActions>
+    <UpdateDatabase id="CQUI_Settings">
```

Comment:
so this saves the plugin settings in a separate file as I understand. worth adding a comment clarifying this (for future developers)

Reply:
I'm not sure where that file is -- as with the change with the honey, it's like 2 hours of futzing around before figuring out the problem is a few lines.  :)  I was looking at other mods that were apparently saving their settings okay and didn't see any files in the mods get updated.
The cache files - like mods.sqlite in the Documents\My Games\Sid Meier's Civilization VI\ gets updated, and that's where - as best I can tell - the data is saved.
Not discounting the possiblity that you're correct, it's likely I just didn't see where the files that get updated happen to live